packages/example-front/src/app/modules/profile/profile.tsx

```diff
@@ -0,0 +1,108 @@
+import { useUserManagement } from '@invorious/access-control-front';
+import { useEffect, useState } from 'react';
+import { IUser } from '../../types/user.interface';
+import ProfileInformation from '../profile-information/profile-information';
+import styles from './profile.module.scss';
+
+export function Profile() {
+  const [updateProfile, setUpdateProfile] = useState(false);
+  const [fetchingUser, setFetchingUser] = useState(false);
+  const [user, setUser] = useState<IUser>();
+  const { requestError, getProfile, update } = useUserManagement({
+    baseURL: 'api/auth',
+    headers: {
+      Authorization: `Bearer ${localStorage.getItem('token')}`,
```

Comment:
I think we create a Auth Helper or services in the front to manage this Headers. we will need to add this token it multiple components...

Reply:
@Miguel-Invorious i think useUserManagement hook should contain the URL of the API, it is his responsability to manage the user data including where it comes from. thoughts???

Reply:
Yeah I think it should do as well, and I'll create a hook to return auth token, something like useBearearToken()